src/memory/spi_mem.c

```diff
@@ -242,6 +242,18 @@ static bool _spi_mem_page_write(uint32_t page_addr, const uint8_t* input)
     // --- Wait for write to end ---
     _spi_mem_wait();
 
+    // --- Check input data has been properly written ---
```

Comment:
I wonder if this slows down writing the BLE firmware in factorysetup. There, it's not needed as we anyway read it back and verify it.

Could you check if if this causes a slowdown? Could also just exclude it there using `#if FACTORYSETUP == 0`

Reply:
Thanks for checking :+1: